src/transactions/types.ts

```diff
@@ -257,6 +257,10 @@ export interface InputGenerateMultiAgentRawTransactionData {
   withFeePayer?: boolean;
 }
 
+export interface OptionalAbi {
```

Comment:
I feel like we can get it into the input generate transaction types

Reply:
Yeah, I had to add it a few places, but I can add it directly.

There's a bit of a weirdness in the input generate transaction types, because there's a duplicate ABI then, but I think I can rework it.